walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -4,6 +4,11 @@
   width: 650px;
 }
 
+#content img {
+  width: 100%;
```

Comment:
Nit: We like to have css styles in alphabetical order so it's easier to find things when they get large

Reply:
Noted, will fix it! 👍